Kubernetes/windows/scripts/powershell/main.ps1

```diff
@@ -49,16 +49,26 @@ function Set-EnvironmentVariables
     if (Test-Path /etc/omsagent-secret/KEY) {
         # TODO: Change to omsagent-secret before merging
         $wsKey =  Get-Content /etc/omsagent-secret/KEY
-    } 
-    
+    }
+
     # Set KEY
     [System.Environment]::SetEnvironmentVariable("WSKEY", $wsKey, "Process")
     [System.Environment]::SetEnvironmentVariable("WSKEY", $wsKey, "Machine")
 
+    $proxy = ""
+    if (Test-Path /etc/omsagent-secret/PROXY) {
+        # TODO: Change to omsagent-secret before merging
+        $wsKey =  Get-Content /etc/omsagent-secret/PROXY
```

Comment:
it should be $proxy not $wskey, else this will break key for clusters with proxy

Reply:
> it should be $proxy not $wskey, else this will break key for clusters with proxy

> Good catch. This needs to be $proxy not $wsKey.